lib/DiscoveryStreamFeed.jsm

```diff
@@ -74,6 +74,7 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
 
     // Persistent cache for remote endpoint data.
     this.cache = new PersistentCache(CACHE_KEY, true);
+    this.locale = Services.locale.appLocaleAsLangTag;
```

Comment:
This should have an observer as the app's primary locale can change, but probably just file a followup bug.

Reply:
Did the old AS have a observer for this, if so, probably easiest if I just connect to that observer here, instead of making a new one.